helper/reports.go

```diff
@@ -0,0 +1,76 @@
+package helper
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+)
+
+func ProcessDownloadReports(fileID int, alwaysApproved bool, generate string, output string) (bool, error) {
```

Comment:
We can remove the return values from this function. If the error happens, we can do `os.Exit(1)`. This is the behaviour in other helper functions as well.

Reply:
Using returning statement in `ProcessDownloadReports` function it is easy to write tests for this function instead of doing os.Exit(1) here.
os.Exit(1) is added in parent method if this fails.